x-pack/legacy/plugins/logstash/public/lib/update_management_sections/update_logstash_sections.js

```diff
@@ -1,22 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { management } from 'ui/management';
-
-export function updateLogstashSections(pipelineId) {
-  const editSection = management.getSection('logstash/pipelines/pipeline/edit');
-  const newSection = management.getSection('logstash/pipelines/pipeline/new');
-
-  newSection.hide();
-  editSection.hide();
-
-  if (pipelineId) {
-    editSection.url = `#/management/logstash/pipelines/pipeline/${pipelineId}/edit`;
-    editSection.show();
-  } else {
-    newSection.show();
-  }
-}
```

Comment:
It appears this behavior was already broken in the legacy plugin, so I removed this code completely. Not sure how useful it was anyways.